### ISSUE TITLE
TextDecoder.decode() TypeError hangs the whole app

### ISSUE THREAD

**Issue by ushuz** [liyasthomas/postwoman#1032]
**Describe the bug**

Postwomen keeps raising following exception on a failed request, and hanging the app. I have to use browser's task manager to kill it. Screenshot below.

```
TypeError: Failed to execute 'decode' on 'TextDecoder': The provided value is not of type '(ArraryBuffer or ArrayBufferView)
```

**To Reproduce**
1. Send an empty POST requests to `https://www.feishu.cn/approval/openapi/v2/approval/get`

**Expected behavior**

Display the error response without hanging the app.

**Screenshots**

![image](https://user-images.githubusercontent.com/1430856/88879358-a0833900-d25c-11ea-9e75-f0fc9a72441f.png)

**Desktop (please complete the following information):**
 - OS: macOS 10.15.6
 - Browser: Edge 84.0.522.48
 - Postwomen Version: v1.9.7

**Comment by liyasthomas** [comment]
@AndrewBastin

**Comment by AndrewBastin** [comment]
This seems to be an issue with the extension. I will open the issue on the extension and I will start working on a fix.

**Comment by AndrewBastin** [comment]
@ushuz the update with the fix is being rolled with the 0.15 browser extension update!

It is live on Firefox right now. Will let you know when it is live on Chrome ^_^
